File: final_project/GroupFinalAPI/Models/customers.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const Customer = new Schema({

    fname:
    {
        type: String,
        required: true
    },
    lname:
    {
        type: String,
        required: true
    },
    email:
    {
        type: String,
        required: true
    },
    phonenumber:
    {
        type: String,
        required: true
    }
    
    
});
mongoose.model('Customer', Customer);

File: final_project/lib/Models/customers.dart

class Customer {
  final String id;
  final String fname;
  final String lname;
  final String email;
  final String phonenumber;
  Customer._(this.id, this.fname, this.lname, this.email, this.phonenumber);
  factory Customer.fromJson(Map json) {
    final id = json[''].replaceAll('ObjectId(\"', '').replaceAll('\")', '');
    final fname = json['fname'];
    final lname = json['lname'];
    final email = json['email'];
    final phonenumber = json['phonenumber'];
    return Customer._(id, fname, lname, email, phonenumber);
  }
}